components/Navbar.tsx

```diff
@@ -48,23 +48,34 @@ export default function Navbar() {
             </a>
           ))}
         </div>
-        <div className="flex flex-1 items-center justify-end gap-x-6">
-          <a
-            href="/#contact"
-            className="rounded-md bg-cyan-600 py-2 px-3 text-sm font-semibold text-white shadow-sm hover:bg-cyan-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-cyan-600"
-          >
-            Contact Me
+        <button className="group flex flex-0  ml-auto items-center justify-end rounded-md text-white bg-cyan-600 py-2 px-3 shadow-sm hover:bg-cyan-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-cyan-600">
+          <a href="/#contact" className="text-sm font-medium">
+            Contact Me{' '}
```

Comment:
I wasn't able to get `space-x-2` to work. Work around is adding this space after the button text.